some improves

ersanyakit opened the pull request:
some improves on project files

Mshuu commented:
Hey mate,

Give me more detail on what you changed and why (the commit looks fine) and I can merge it in. Need more details though.

ersanyakit commented:
> Hey mate,
> 
> Give me more detail on what you changed and why (the commit looks fine) and I can merge it in. Need more details though.
I cleaned home directory. Moved images/css to assets folder.
Some small improves on html side...
added Manifest file.. I added version number to some css/js files...
Also small changes on mainjs.
